rts/Lua/LuaHandle.cpp

```diff
@@ -172,6 +172,10 @@ void CLuaHandle::KillLua(bool inFreeHandler)
 	if (inFreeHandler)
 		Shutdown();
 
+	if(rmlui) {
+		RmlGui::Reload();
```

Comment:
shouldn't this be some sort of shutdown and not reload?

Reply:
Completed. On luaui shutdown we will remove the lua rml plugin and all contexts/documents created from lua. This allows the engine to continue to use rml for it's interface.